### ISSUE TITLE
Apide Vietnam - Fullstack Web Developer - Saigon - FT

### ISSUE THREAD

**Issue by RecruiterMaiNgo** [awesome-jobs/vietnam#251]
## A 100% foreign owned company

You are passionate about software development and doing it the right way? Awesome so are we, our customers deserve the best they can get!

You embrace changes and see it as a necessity in order to make great software? Yup so do we, and we love the agile approach!

Join Apide Vietnam as we set out on our mission to become the best software development company in Việt Nam!
## Location
- Saigon, Vietnam
## Salary Expectation
- Negotiable
### Active projects
- Checking stories before starting on a  project
- Writing code and automated tests as  required
- Requirements analysis with  clients
- Tech breakdowns with the  team
- Using the app and manual  testing
- Deploying code, building  servers
- Working with other agencies to e.g. agree on APIs, comment on   designs
- End of project process (demos &  retrospectives)
### Training and community building
- Internal mentoring / Present at community  events
### Commercial criteria
- Able to communicate clearly with both non­technical clients and hardcore developers
- Happy to take responsibility (and authority) for every stage of a   project
- Reliability and attention to  detail

**Cultural criteria**
- Excited by new developments on the web, and constantly   learning
- Pragmatic – motivated by project success as well as code   beauty
- Opinionated about technology and able to argue a case intelligently and respectfully
### Must Have
- Code: Can write Ruby (on Rails), HTML, JavaScript and CSS  proficiently
- Learning: Can get up to speed on new technologies quickly, and independently by using online resources
- Independence & collaboration: Can work individually or as part of a technical team depending on the project
- Communication: Can extract a clear set of requirements from a non­technical client through careful questioning, and communicate proficiently in English, both orally  and in writing
- Product design: Can adapt requirements to something technically feasible and commercially successful
- Databases: Can use MySQL and query the database directly using SQL, furthermore it’s an advantage to be familiar with MongoDB and other non­SQL databases
### Important
- JavaScript: Can write complex, structured JavaScript proficiently, knowledge about AngularJS is an advantage
- HTML & CSS: Can do HTML markup and styling based on a design provided by clients
- Ruby: Code is modular and existing gems are used in favor of home cooked solutions
### Nice to have
- Server: Can deploy to, and manage a Linux  box
- UX: Can create efficient user  flows
- Visuals: Can create visual  designs
- Scrum: Can write user stories, plan sprints, draw burndowns, and run   demos
- Training: Can train others through blogging, mentoring, and   presenting
- Knowledge about other programming languages + frameworks: Can program in PHP with a framework such as Symphony or C# with MVC   .NET
- Experience with WordPress customization such as theme   development
## Why you love working here
- Work in an amazing start-up company in Vietnam with very open and friendly working environment
- Develop "sense of ownership" of product company in an Out-sourcing company
## Contact
- Ms Mai Ngo - HR Recruiter: 01243 894 915
- Skype: mai.ngo91
- Email: mai@apide.vn
